x-pack/test/functional/page_objects/enterprise_search.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export function EnterpriseSearchPageProvider({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common']);
+
+  return {
+    async navigateToPage() {
+      return await PageObjects.common.navigateToApp('enterprise_search');
```

Comment:
Just curious, how will this be affected (if at all) when we're registering 2-3 separate plugins (App Search, Workplace Search, and Enterprise Search)?

Reply:
Ahh disregard my question here, I see that we use [`.navigateToApp('app_search')`](https://github.com/constancecchen/kibana/pull/6/files#diff-a61f4f16fea7f280127bf23d9430acfcR15)